plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageSettings.java

```diff
@@ -425,9 +422,13 @@ private CTabItem createPageTab(@NotNull IDialogPage page, int index) {
         item.setToolTipText(page.getDescription());
 
         if (page.getControl() == null) {
-            final Composite placeholder = new Composite(tabFolder, SWT.NONE);
+            final ScrolledComposite sc = new ScrolledComposite(tabFolder, SWT.BORDER | SWT.H_SCROLL);
+            sc.setExpandHorizontal(true);
+            sc.setExpandVertical(true);
+            final Composite placeholder = new Composite(sc, SWT.NONE);
```

Comment:
I don't think you need a placeholder here. Put controls directly inside your `ScrolledComposite`

Reply:
Just for use 'setContent' - require to specify scrollable content